### PR TITLE
Updated CNI ports per Slack feedback

### PR DESCRIPTION
**Pull request by jtravee**
Related PR #2381 

Updating port requirements per @manuelbuil from feedback in Slack.

### CODE REVIEW

**Review by briandowns — APPROVED** [review]

File: docs/install/requirements.md [hunk 1]
```diff
@@ -76,13 +76,14 @@ If you wish to utilize the metrics server, you will need to open port 10250 on e
...
-| UDP | 8742 | RKE2 server and agent nodes | Cilium VXLAN
+| UDP | 8472 | RKE2 server and agent nodes | Cilium VXLAN
 | TCP | 4240 | RKE2 server and agent nodes | Cilium health checks
 | ICMP | 8/0 | RKE2 server and agent nodes | Cilium health checks
 | TCP | 179 | RKE2 server and agent nodes | Calico CNI with BGP
 | UDP | 4789 | RKE2 server and agent nodes | Calico CNI with VXLAN
 | TCP | 5473 | RKE2 server and agent nodes | Calico CNI with Typha
-| UDP | 4789 | RKE2 server and agent nodes | Canal CNI with VXLAN
+| UDP | 8472 | RKE2 server and agent nodes | Canal CNI with VXLAN
+| TCP | 9099 | RKE2 server and agent nodes | Canal health checks
```
Comment (brandond) [inline docs/install/requirements.md:86]:
```suggestion
| TCP | 9099 | RKE2 server and agent nodes | Canal CNI health checks
```

File: docs/install/requirements.md [hunk 2]
```diff
@@ -76,13 +76,14 @@ If you wish to utilize the metrics server, you will need to open port 10250 on e
 | TCP | 2379 | RKE2 server nodes | etcd client port
 | TCP | 2380 | RKE2 server nodes | etcd peer port
 | TCP | 30000-32767 | RKE2 server and agent nodes | NodePort port range
-| UDP | 8742 | RKE2 server and agent nodes | Cilium VXLAN
+| UDP | 8472 | RKE2 server and agent nodes | Cilium VXLAN
 | TCP | 4240 | RKE2 server and agent nodes | Cilium health checks
 | ICMP | 8/0 | RKE2 server and agent nodes | Cilium health checks
```
Comment (brandond) [inline docs/install/requirements.md:81]:
```suggestion
| UDP | 8472 | RKE2 server and agent nodes | Cilium CNI VXLAN
| TCP | 4240 | RKE2 server and agent nodes | Cilium CNI health checks
| ICMP | 8/0 | RKE2 server and agent nodes | Cilium CNI health checks
```

**Review by brandond — COMMENTED** [review]
Nit: lets be consistent about putting "CNI" in the description

File: docs/install/requirements.md (hunk 2 above)
Comment (jtravee) [reply docs/install/requirements.md:81]: Agreed, ty!

**Review by brandond — APPROVED** [review]

**Pull request merged**